plugins/serendipity_plugin_syndication/serendipity_plugin_syndication.php

```diff
@@ -1,9 +1,5 @@
 <?php
 
-if (IN_serendipity !== true) {
```

Comment:
Small remark: I don't think that should be removed here, as it is present in the other plugins.